server/src/main/java/org/elasticsearch/index/mapper/ObjectMapper.java

```diff
@@ -403,6 +403,10 @@ public ObjectMapper.Builder newBuilder(IndexVersion indexVersionCreated) {
         return builder;
     }
 
+    public ObjectMapper withoutMappers() {
```

Comment:
Does this here and in the subclasses need to be public? Can you also use the `@Override` annotation in the subclasses that override it?